protocols.md

```diff
@@ -0,0 +1,73 @@
+# Protocols
+
+> Protocols are the "attitudes" or "the manner in which one approaches each and
+every element in our space." They are different than pure rules or
+instructions; they are ways that we establish order and maintain practices
+across our group [...] -- They define the way we ought to proceed or behave in
+different situations. As such, they are normative, or premised on values,
+morals, and an idea of how things ought to be done. They are a manifestation of
+our values.
+>
+> -- [Lab Book, Civic Laboratory for Environmental Action Research (CLEAR)][clear-lab-book]
+
+## Scheduling a meeting
+
+Note: Consider whether this meeting might be a global interest to other
+  members. Skip below steps at your discretion for low-stakes topics.
+
+- Try to announce your intention to schedule a meeting on a topic, and ask
+  who's interested to participate. Allow a day for people to respond.
+- Consider who might be most interested, and what timing might work for
+  them.
+  - Check the [monthly `availability` calendar for travel/vacation plans][availability].
+  - Check everyone's [weekly availability `schedules`][schedules].
+    for them.
+- If 2-3 candidate timeslots feel obvious from looking at internal
+  availabilities:
+  - Reserve them all in [Hypha calendar][calendar] immediately, for example: `HOLD:
+    Infrastructure Meeting (Option 1)`
+  - Ask in chat whether anyone who wants to attend has any blocks on any
+    of the options.
+- If short-notice or speed of booking is important:
+  - Claim an open HOLD [:information_source:](/meetings.md#holds) slot.
+    - If time permits or topic isn't an all-hands matter, consider
+      leaving the hold available for someone else.
+    - If claiming a hold, edit the calendar immediately, and announce
+      meeting in chat.
+- If your must find a new timeslot:
+  - Create a [new When2Meet poll](https://www.when2meet.com/).
```

Comment:
Maybe say "Create a new poll" (we currently use [When2Meet](#))

Reply:
Good call. Committed locally.